Core/scripts/dirac-install.py

```diff
@@ -318,7 +318,12 @@ def __loadCFGFromURL( self, urlcfg, checkHash = False ):
       self.__cfgCache[ urlcfg ] = cfg
       return S_OK( cfg )
     try:
-      md5Data = urlretrieveTimeout( urlcfg[:-4] + ".md5", timeout = 60 )
+      md5path = urlcfg[:-4] + ".md5"
+      if os.path.exists( md5path ):
+        with open( md5path, 'r' ) as md5File:
```

Comment:
Actually, no, this file should be possibly read by python 2.6, so don't use `woth`